src/Microsoft.Identity.Web/WebApiServiceCollectionExtensions.cs

```diff
@@ -121,6 +122,8 @@ public static class WebApiServiceCollectionExtensions
 
             services.Configure<JwtBearerOptions>(jwtBearerScheme, options =>
             {
+                options.Events ??= new JwtBearerEvents();
+
```

Comment:
nit: spacing

Reply:
What do you mean?

Reply:
the space between ??=

Reply:
There shouldn't be one. It's [compound operator](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/null-coalescing-operator).

Reply:
Hah, never used this operator before. It's nice to see smth new :)